plugin/helpers.go

```diff
@@ -85,7 +83,7 @@ type Helpers interface {
 	// Use this URL to link to assets from the webapp, or for third-party integrations with your plugin.
 	//
 	// Minimum server version: 5.2
-	GetPluginAssetURL(pluginID, asset string) (*url.URL, error)
+	GetPluginAssetURL(pluginID, asset string) (string, error)
```

Comment:
0/5, but why the change to `string` here from `url.URL`?

Reply:
@lieut-data `path.Join()` would require a `string`, I also figured out in the course of working on the feedback that it would easier and also for testing to return a string since what we care about is the `URL`, I didn't figure that initially when I worked on it. Is there any concern with the new implementation?

Reply:
Sounds reasonable, thanks for clarifying @Lumexralph!